GitTfs/Core/IGitRepository.cs

```diff
@@ -16,7 +16,7 @@ public interface IGitRepository : IGitHelpers
         IGitTfsRemote ReadTfsRemote(string remoteId);
         IGitTfsRemote CreateTfsRemote(RemoteInfo remoteInfo, string autocrlf = null, string ignorecase = null);
         void DeleteTfsRemote(IGitTfsRemote remoteId);
-        IEnumerable<string> GetGitRemoteBranches(string gitRemote);
+        IEnumerable<string> GetGitRepositoryBranchRemotes(string gitRemote);
```

Comment:
I wouldn't change this method name--it's on an interface. Do you know who's all implementing this interface? Is it only Git-TFS libraries that implement this interface? You can't be sure, because the interface is `public`.

I really think this needs to be reverted.